entities/build.gradle

```diff
@@ -3,6 +3,12 @@ plugins {
 }
 dependencies {
     implementation(project(':common'))
+
     implementation('com.fasterxml.jackson.core:jackson-databind:2.14.2')
+    implementation('javax.validation:validation-api:2.0.1.Final')
```

Comment:
since javax is deprecated, should we use jakarta validation package?

Reply:
done